src/ui/display.c

```diff
@@ -74,99 +69,186 @@ UX_STEP_CB(ux_display_reject_step,
                "Reject",
            });
 
-// FLOW to display address and BIP32 path:
```

Comment:
I think we should keep this explanation. It seems useful to understand what's happening.